.travis.yml

```diff
@@ -20,8 +20,8 @@ virtualenv:
 install:
   - git clone https://github.com/OCA/reporting-engine.git ${HOME}/reporting-engine -b ${VERSION}
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
-  - sudo wget http://downloads.sourceforge.net/project/wkhtmltopdf/archive/0.12.1/wkhtmltox-0.12.1_linux-precise-amd64.deb -P /tmp/
-  - sudo dpkg -i /tmp/wkhtmltox-0.12.1_linux-precise-amd64.deb
+  - wget http://download.gna.org/wkhtmltopdf/0.12/0.12.1/wkhtmltox-0.12.1_linux-precise-amd64.deb
+  - sudo dpkg -i wkhtmltox-0.12.1_linux-precise-amd64.deb
```

Comment:
well, not that I know of: you need root to install a package. The alternative is building wkhtmltopdf from source, which I don't wish to write the code for right now.

This will be a show stopper for the mass commit I'm planning on that repository, but it just means that this branch will stay on the VM builds.